src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.model;
+
```

Comment:
Don't begin class or method implementation with empty line. Remove the redundant empty line at the beginning of the class definition.